feat(applications-veeam-centreon-monitoring-agent): added new connector for Veeam using Centreon Monitoring Agent (#4112)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-veeam-centreon-monitoring-agent.md

</TabItem>
</Tabs>

### Using a service template provided by the connector

1. If you have used a host template and checked **Create Services linked to the Template too**, the services linked to the template have been created automatically, using the corresponding service templates. Otherwise, [create manually the services you want](/docs/monitoring/basic-objects/services) and apply a service template to them.
2. Fill in the macros you want (e.g. to change the thresholds for the alerts). Some macros are mandatory (see the table below).

<Tabs groupId="sync">
<TabItem value="Job-Status" label="Job-Status">

| Macro           | Description                                                                                                                                                               | Default value                                           | Mandatory   |
|:----------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------|:-----------:|

> | FILTERENDTIME   | Filter job with end time greater than current time less value in seconds                                                                                                  | 86400                                                   |             |
> | FILTERNAME      | Filter job name (can be a regexp)                                                                                                                                         |                                                         |             |
> | FILTERSTARTTIME | Filter job with start time greater than current time less value in seconds                                                                                                |                                                         |             |

Author: ```suggestion
| FILTERENDTIME   | Tolerance value in seconds, to avoid skipping jobs whose end time is earlier than the current time                                                                                               | 86400                                                   |             |
| FILTERNAME      | Filter job name (can be a regexp)                                                                                                                                         |                                                         |             |
| FILTERSTARTTIME | Tolerance value in seconds, to avoid skipping jobs whose start time is earlier than the current time                                                                                              |                                                         |             |
```

| --ps-exec-only      |   Print powershell output.                                                                                                                                                                                                                          |
| --filter-name       |   Filter job name (can be a regexp).                                                                                                                                                                                                                |
| --exclude-name      |   Exclude job name (regexp can be used).                                                                                                                                                                                                            |
| --filter-type       |   Filter job type (can be a regexp).                                                                                                                                                                                                                |

> | --filter-start-time |   Filter job with start time greater than current time less value in seconds.                                                                                                                                                                       |
> | --filter-end-time   |   Filter job with end time greater than current time less value in seconds (default: 86400).                                                                                                                                                        |

Author: ```suggestion
| --filter-start-time |   Tolerance value in seconds, to avoid skipping jobs whose start time is earlier than the current time.                                                                                                                                                                       |
| --filter-end-time   |   Tolerance value in seconds, to avoid skipping jobs whose end time is earlier than the current time (default: 86400).      |
```

---
id: applications-veeam-centreon-monitoring-agent
title: Veeam Centreon Monitoring Agent
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Connector dependencies

The following monitoring connectors will be installed when you install the **Veeam Centreon Monitoring Agent** connector through the
**Configuration > Monitoring Connector Manager** menu:
* [Base Pack](./base-generic.md)

## Pack assets

### Templates

> The Monitoring Connector **Veeam Centreon Monitoring Agent** brings 2 host templates:

Reviewer: ```suggestion
The Monitoring Connector **Veeam Centreon Monitoring Agent** brings one host template:
```

* **App-Veeam-Centreon-Monitoring-Agent-custom**

> * **generic-passive-host-custom**

Reviewer: ```suggestion
```

|:-------------------------------------------------|:------|
| status                                           | N/A   |
| *repositories*~repository.space.usage.bytes      | B     |
| *repositories*~repository.space.free.bytes       | B     |
| *repositories*~repository.space.usage.percentage | %     |

</TabItem>
<TabItem value="Tape-Jobs" label="Tape-Jobs">

| Name                 | Unit  |
|:---------------------|:------|
| tapejobs.total.count | count |

</TabItem>
<TabItem value="Vsb-Jobs" label="Vsb-Jobs">

| Name                            | Unit  |
|:--------------------------------|:------|
| sure_backup.jobs.detected.count | count |
| sure_backup.jobs.success.count  | count |
| sure_backup.jobs.failed.count   | count |
| sure_backup.jobs.warning.count  | count |
| status                          | N/A   |

</TabItem>
</Tabs>

## Prerequisites

### Network flow

Only one TCP flow must be open from the host to the poller.

| Source         | Destination | Protocol | Port | Purpose                                          |
|----------------|-------------|----------|------|--------------------------------------------------|

> | Monitored host | Collecteur  | TCP      | 4317 | Configuration retrieval OpenTelemetry data flow. |

Reviewer: ```suggestion
| Monitored host | Collecteur  | TCP      | 4317 | Configuration retrieval, and OpenTelemetry data flow. |
```

</TabItem>
</Tabs>

### Using a service template provided by the connector

1. If you have used a host template and checked **Create Services linked to the Template too**, the services linked to the template have been created automatically, using the corresponding service templates. Otherwise, [create manually the services you want](/docs/monitoring/basic-objects/services) and apply a service template to them.
2. Fill in the macros you want (e.g. to change the thresholds for the alerts). Some macros are mandatory (see the table below).

<Tabs groupId="sync">
<TabItem value="Job-Status" label="Job-Status">

| Macro           | Description                                                                                                                                                               | Default value                                           | Mandatory   |
|:----------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------|:-----------:|

> | FILTERENDTIME   | Filter job with end time greater than current time less value in seconds                                                                                                  | 86400                                                   |             |

Reviewer: ?

Author: https://docs.centreon.com/fr/pp/integrations/plugin-packs/procedures/applications-veeam-nrpe/
![image](https://github.com/user-attachments/assets/dbb52069-69c3-456e-94f4-100b7da6e923)
